services/wallet/thirdparty/collectible_types.go

```diff
@@ -152,6 +152,12 @@ type CollectionData struct {
 	ImageURL     string                `json:"image_url"`
 	ImagePayload []byte
 	Traits       map[string]CollectionTrait `json:"traits"`
+	Socials      CollectionSocials          `json:"socials"`
```

Comment:
same as the previous comment, make this a pointer since only some providers give us full Socials info.